schema/crdb/4.0.1/up.sql

```diff
@@ -0,0 +1,24 @@
+-- CRDB documentation recommends the following:
```

Comment:
FWIW in my original recommendation, I figured it would have been the following, within a *single* file named `4.0.0/up.sql`, but what you have done works too.

```sql
BEGIN;
SELECT CAST(
    IF(
        (
            SELECT version = '3.0.3' and target_version = '4.0.0'
            FROM omicron.public.db_metadata WHERE singleton = true
        ),
        'true',
        'Invalid starting version for schema change'
    ) AS BOOL
);

ALTER TYPE omicron.public.service_kind ADD VALUE IF NOT EXISTS 'clickhouse_keeper';
COMMIT;

BEGIN;
SELECT CAST(
    IF(
        (
            SELECT version = '3.0.3' and target_version = '4.0.0'
            FROM omicron.public.db_metadata WHERE singleton = true
        ),
        'true',
        'Invalid starting version for schema change'
    ) AS BOOL
);

ALTER TYPE omicron.public.dataset_kind ADD VALUE IF NOT EXISTS 'clickhouse_keeper';
COMMIT;
```

Reply:
thnx!